source/Willow/Willow.class.st

```diff
@@ -0,0 +1,63 @@
+"
+I'm a shared pool providing access to constants and classes defined by the lilbrary
+"
+Class {
+	#name : #Willow,
+	#superclass : #SharedPool,
+	#classVars : [
+		'Classification',
+		'Constants'
+	],
+	#pools : [
+		'RenoirSt'
+	],
+	#category : #'Willow-Frontend'
+}
+
+{ #category : #accessing }
+Willow class >> classification [
+
+	^ Classification
+]
+
+{ #category : #accessing }
+Willow class >> constants [
+
+	^ Constants
+]
+
+{ #category : #initialization }
+Willow class >> initialize [
+
+	<ignoreForCoverage>
+	self initializeConstants.
+	self initializeClasses
+]
+
+{ #category : #'private - initialization' }
+Willow class >> initializeClasses [
+
+	<ignoreForCoverage>
+	Classification := Namespace new.
+	Classification
+		bind: #fadingNotification to: 'willow-fading-notification';
+		bind: #loading to: 'willow-loading';
+		bind: #dialogContainer to: 'willow-dialog-container';
+		bind: #willow to: 'willow';
+		bind: #disabledComponent to: 'willow-disabled-component'
+]
+
+{ #category : #initialization }
+Willow class >> initializeConstants [
+
+	<ignoreForCoverage>
+	Constants := Namespace new.
+	Constants bind: #css to: CssConstants.
```

Comment:
I guess binding CssConstants to css connects Willow with Renoir? It might be easier to let whoever wants to use one or the other to explicitly reference the shared pool.

Reply:
I've made that link because you can access the constants without having to include the pool in the command building block.